test/event_emitter.js

```diff
@@ -0,0 +1,22 @@
+import { EventEmitter } from '../src/util/event_emitter';
+
+describe('EventEmitter', function() {
+  beforeEach(function() {
+    this.emitter = new EventEmitter();
```

Comment:
personally I'm against using `this` in unit tests, I would rather define the var outside of `beforeEach` and initialise it inside.